### DISCUSSION TITLE
Masque 1 Pixel Black Border

### DISCUSSION THREAD

**Discussion by SuperNoodlePlays** [SFX-WoW/Masque#178] [Issues] [answered]
After installing masque I get this one pixel black border around my weakauras. Any clue how to disable it?

![image](https://user-images.githubusercontent.com/37938058/39405555-4eccd4d8-4ba7-11e8-8130-17775840b96f.png)

**Comment by StormFX** [comment]
Check various settings in WA. This isn't a Masque issue.

**Comment by Marving1023** [comment, marked as answer]
Try this https://github.com/SFX-WoW/Masque/discussions/176.

**Comment by StormFX** [comment]
This issue is due to improper settings in WeakAuras and/or ElvUI. Closing.
